### ISSUE TITLE
Does this tool just call speedtest.net?

### ISSUE THREAD

**Issue by nbro** [nelsonjchen/speedtest-rs#114]
I know that there are several tools around that just make a request to the famous speedtest website or maybe call the command-line tool. Without looking at the code and reading too much around, it seems to me that this tool does the same. Is this correct?

**Comment by nelsonjchen** [comment]
Yeah, it is a clone of speedtest-cli which did that. I think it was based off the logic of the reverse-engineered flash applet.

**Comment by nelsonjchen** [comment]
Eh, I wouldn't even call this a package. I actually don't even want it as a package. It is caveat emptor if you use it as a package. Also, terminology wise, people call this sort of thing a crate in Rust parlance.

I did expose it as a crate by some demand, should anyone want to try to use it's "API", which will forever stay at 0.0.x-ish . But it should just be considered a tool. At the time, speedtest-cli was also written in a manner that wasn't "library" friendly either. I didn't copy that, but I didn't go out of my way to go away from that either. 

I'm sure there are many other fun rust tools, but I haven't really touched this project in years. I'm not even quite sure why there's a surge of interest in this tool right now. At the time I wrote this, I was looking to get a non-garbage collected speed test tool running on an ARMv5TEJ appliance and rewriting the slow Python to Rust was just the simplest way to do it. That was a long, long time ago. I haven't kept up.
